ppml/services/bigdl-spark-operator/docker/Dockerfile

```diff
@@ -0,0 +1,151 @@
+ARG SPARK_VERSION=3.1.2
+ARG HADOOP_VERSION=2.7
+ARG SPARK_HOME=/opt/spark
+ARG JDK_VERSION=8u192
+ARG JDK_URL=your_jdk_url
+ARG BIGDL_VERSION=2.1.0-SNAPSHOT
```

Comment:
2.2.0-SNAPSHOT is incompatible with python dependencies in env.tar.gz, and will cause a spark shutdown.
2.1.0 needs to be further tested.

Reply:
> 2.2.0-SNAPSHOT is incompatible with python dependencies in env.tar.gz, and will cause a spark shutdown. 2.1.0 needs to be further tested.

Why incompatible? Did you provide the conda archive with bigdl-2.1.0 to users instead of packing it themselves?

Reply:
> 2.2.0-SNAPSHOT is incompatible with python dependencies in env.tar.gz, and will cause a spark shutdown. 2.1.0 needs to be further tested.

We have stopped updating and maintaining 2.1.0-snapshot since Sep, guess we better use 2.1.0 officially instead😄.

Reply:
The driver&executor I used in test is `bigdl-k8s-spark-3.1.2:2.1.0-SNAPSHOT-20220509` as presented in the wiki.

As a result, the BIGDL_VERSION in operator should also be 2.1.0-SNAPSHOT, otherwise an error like will occur:
`  File "/bigdl2.0/data/le/python_env/lib/python3.7/site-packages/bigdl/dllib/utils/common.py", line 107, in __init__
    self.value.append(getattr(jclass, "ofFloat")())
TypeError: 'JavaPackage' object is not callable`

Le told me that this may partly because the jars in `env.tar.gz ` is incompatible with 2.2.0-SNAPSHOT. 

I am going try the 2.1.0 version. Is there a 2.1.0 driver&executor image to replace the SNAPTHOT one?

Reply:
seems we need to new image to test. Try `intelanalytics/bigdl-k8s:2.1.0`, that's a stable image for 2.1.0 official release.